src/item.h

```diff
@@ -495,15 +495,24 @@ class ItemAttributes
 			return false;
 		}
 
+		const static uint32_t intAttrType = ITEM_ATTRIBUTE_ACTIONID | ITEM_ATTRIBUTE_UNIQUEID | ITEM_ATTRIBUTE_DATE
+			| ITEM_ATTRIBUTE_WEIGHT | ITEM_ATTRIBUTE_ATTACK | ITEM_ATTRIBUTE_DEFENSE | ITEM_ATTRIBUTE_EXTRADEFENSE
+			| ITEM_ATTRIBUTE_ARMOR | ITEM_ATTRIBUTE_HITCHANCE | ITEM_ATTRIBUTE_SHOOTRANGE | ITEM_ATTRIBUTE_OWNER
+			| ITEM_ATTRIBUTE_DURATION | ITEM_ATTRIBUTE_DECAYSTATE | ITEM_ATTRIBUTE_CORPSEOWNER | ITEM_ATTRIBUTE_CHARGES
+			| ITEM_ATTRIBUTE_FLUIDTYPE | ITEM_ATTRIBUTE_DOORID;
+		const static uint32_t stringAttrType = ITEM_ATTRIBUTE_DESCRIPTION | ITEM_ATTRIBUTE_TEXT | ITEM_ATTRIBUTE_WRITER
+			| ITEM_ATTRIBUTE_NAME | ITEM_ATTRIBUTE_ARTICLE | ITEM_ATTRIBUTE_PLURALNAME;
+		const static uint32_t customAttrType = ITEM_ATTRIBUTE_CUSTOM;
```

Comment:
This is not necessary, I doubt there will be more custom attribute types.

Reply:
Actually, stringAttrType and intAttrType should be plural: `stringAttributeTypes` and `intAttributeTypes` (yes, no need to abbreviate attr either).

Reply:
Renamed to plural and full 'attribute' name. Removed 'custom' constant.